example/main.go

```diff
@@ -2,59 +2,27 @@ package main
 
 import (
 	"context"
-	"flag"
 	"fmt"
-	"log"
 	"os"
 	"strings"
 
 	"github.com/wapc/wapc-go"
-	"github.com/wapc/wapc-go/engines/wasmtime"
```

Comment:
I resisted making engine_xx.go which is a way to have build tags decide which engine to use. that could happen later if necessary.